Resources/Server Info/Guidebook/Science/APE.xml

```diff
@@ -0,0 +1,20 @@
+<Document>
+# A.P.E.
+
+The Anomalous Particle Emitter is a machine used to interface with anomalies. It is the primary way of manipulating the stats of an anomaly.
+
+<Box>
+<GuideEntityEmbed Entity="MachineAPE"/>
+</Box>
+
+The A.P.E. can shoot 3 different kinds of anomalous particles: Delta, Epsilon, and Zeta. Each particle types has certain effects:
```

Comment:
I was alwasy taught to write numbers from one to ten fully, and I think it reads a bit better too.